### PR TITLE
action: update specs

### PR DESCRIPTION
**Pull request by v1v**
### What

Use GitHub actions in addition to the [`updatecli`](https://www.updatecli.io/) to bump the json schemas created in `ecs-logging`.

It runs on a daily basis and create a Pull Request if new changes exist.

### Test

https://github.com/elastic/ecs-dotnet/pull/250 was created with this setup

### Tasks

- [x] Create description with the commit.

Create PRs with the description pointing to the original change in the ecs-logging, similarly done in https://github.com/elastic/ecs-dotnet/pull/235

### CODE REVIEW

File: .github/workflows/update-specs.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+# Send PRs to the subscribed ECS Agents if the spec files (JSON) are modified
+name: Update specs
+
+on:
+  schedule:
```
Comment (v1v) [inline .github/workflows/update-specs.yml:6]:
```suggestion
  workflow_dispatch:
  schedule:
```

File: .ci/update-specs.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+actions:
+  ecs-dotnet:
+    kind: github/pullrequest
+    scmid: githubConfig
+    sourceid: sha
+    spec:
+      automerge: false
+      labels:
+        - dependencies
+      title: 'synchronize ecs-logging spec'
```
Comment (reakaleek) [inline .ci/update-specs.yml:25]: Is this actually working for you? 😲

**Review by reakaleek — APPROVED** [review]

File: .ci/update-specs.yml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      key: ".[0].sha"
+
+conditions:
+  spec.json-HasSameContentAsSource:
+    name: has specs same content?
+    kind: file
+    sourceid: spec.json
+    spec:
+      file: tests/Elastic.CommonSchema.Tests/Specs/spec.json
+
```
Comment (v1v) [inline .ci/update-specs.yml:57]:
```suggestion
```

File: .ci/update-specs.yml (hunk 2 above)
Comment (v1v) [reply .ci/update-specs.yml:25]: I'm using `dry-run` in my local, let me try with my fork

Comment (v1v) [reply .ci/update-specs.yml:25]: https://github.com/v1v/ecs-dotnet/pull/1 nop! 

But by changing the `target` name:
- https://github.com/v1v/ecs-dotnet/pull/2

```diff
 targets:
   spec.json-update:
-    name: Update spec.json
+    name: 'synchronize ecs-logging spec'
     kind: file
     sourceid: spec.json
     scmid: githubConfig
```

File: .ci/update-specs.yml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  sha:
+    name: Get commit
+    kind: json
+    spec:
+      file: 'https://api.github.com/repos/elastic/ecs-logging/commits?path=spec%2Fspec.json&page=1&per_page=1'
+      key: ".[0].sha"
+
+targets:
+  spec.json-update:
+    name: Update spec.json
```
Comment (v1v) [inline .ci/update-specs.yml:52]:
```suggestion
    name: 'synchronize ecs-logging spec'
```

File: .ci/update-specs.yml (hunk 2 above)
Comment (reakaleek) [reply .ci/update-specs.yml:25]: Interesting. Thanks for checking.

Comment (reakaleek) [reply .ci/update-specs.yml:25]: Interesting. Thanks for checking.

Comment (reakaleek) [reply .ci/update-specs.yml:25]: Interesting. Thanks for checking.

**Review by Mpdreamz — APPROVED** [review]

**Pull request merged**